src/game/data.py

```diff
@@ -40,3 +40,79 @@ class GameTurn:
     def __init__(self):
         self.player_one_active = True
         self.selected_piece = 0
+
+
+class GameState:
+    """Definition of all data of the game at an instant:
+    - grid
+    - remaining_pieces
+    - game_turn"""
```

Comment:
Can you add an example of instanciation in the annotation ?
It would be more clear I think